h/schemas/base.py

```diff
@@ -12,6 +12,9 @@
 from pyramid.session import check_csrf_token
 
 
+PROPERTIES = "properties"
```

Comment:
Any particular reason this is a constant? I thought you preferred to avoid constants whose names are basically their values?

Reply:
My original thought was if we used a different schema validator it might change but I don't think that's gonna happen anymore so I'll remove it.